keras_cv/layers/preprocessing/random_cutout.py

```diff
@@ -155,6 +151,16 @@ def __init__(
         self.fill_mode = fill_mode
         self.fill_value = fill_value
         self.seed = seed
+        self.num_cutouts = num_cutouts
+        if isinstance(num_cutouts, tuple):
+            self.num_cutouts = tf.random.uniform(
+                shape=(1,),
+                minval=self.num_cutouts_lower,
+                maxval=self.num_cutouts_upper + 1,
+                dtype=tf.int32,
+                seed=self.seed,
+            )
+            self.num_cutouts = self.num_cutouts[0]
```

Comment:
This doesn't look right - it looks like you always assign to self.num_Countouts[0]